<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateConnectionDialog</class>
 <widget class="QDialog" name="CreateConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create connection</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Host name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="inputHostName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Port number</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="inputPortNumber">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>3306</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>User name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="inputUserName"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="inputPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="inputDatabase"/>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="gtk-cancel" resource="../res/paomedia-small-n-flat/iconset.qrc">
         <normaloff>:/svg/sign-ban</normaloff>:/svg/sign-ban</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset theme="document-save" resource="../res/paomedia-small-n-flat/iconset.qrc">
         <normaloff>:/svg/floppy</normaloff>:/svg/floppy</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="inputDisplayName"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputDisplayName</tabstop>
  <tabstop>inputHostName</tabstop>
  <tabstop>inputPortNumber</tabstop>
  <tabstop>inputUserName</tabstop>
  <tabstop>inputPassword</tabstop>
  <tabstop>inputDatabase</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonSave</tabstop>
 </tabstops>
 <resources>
  <include location="../res/paomedia-small-n-flat/iconset.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>CreateConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
